jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartConfiguration.java

```diff
@@ -160,37 +171,42 @@ protected void configure(QuickStartGeneratorConfiguration generator, WebAppConte
     @Override
     public void configure(WebAppContext context) throws Exception
     {
-        if (!_quickStart)
+        if (!isQuickStartEnabled(context))
         {
-            super.configure(context);
+            return;
         }
-        else
-        {
-            //add the processor to handle normal web.xml content
-            context.getMetaData().addDescriptorProcessor(new StandardDescriptorProcessor());
 
-            //add a processor to handle extended web.xml format
-            context.getMetaData().addDescriptorProcessor(new QuickStartDescriptorProcessor());
+        //add the processor to handle normal web.xml content
```

Comment:
I'd leave out these changes and the ones following.

Reply:
calling `super.configure(context);` is a no-op.
The rest of the changes in this location are simplifying the if/else logic, that's all.